_i18n/en.yml

```diff
@@ -173,6 +173,8 @@ documentation:
       title: "Using external storage"
     synchronization:
       title: "How to synchronize between devices"
+    statistics:
+      title: "About listening statistics"
```

Comment:
I would remove the word "about" in front of the title. We could add that to every single title, and it's redundant.